README.md

```diff
@@ -87,9 +107,9 @@ Bugs, issues and contributions can be requested to both the mailing list or the
 ## Requirements & dependencies
 
 * Python v2.7+
-* Python v3.4+ ***[STALLED] (experimental: [see known issues](https://github.com/OfflineIMAP/offlineimap/issues?q=is%3Aissue+is%3Aopen+label%3APy3))***
+* Python v3.4+ ***(experimental: [see known issues](https://github.com/OfflineIMAP/offlineimap/issues?q=is%3Aissue+is%3Aopen+label%3APy3))***
```

Comment:
It IS stalled. Nobody is working on this anymore. Please, discard this hunk.

Reply:
OK.

Reply:
On second thought... Shouldn't we keep the "[STALLED]" label for `python3`. 
* Because Travis is testing the `python3` environment. And it's passing.  Even though, it's "cheating" by shebang `python2`.  It will usually run on a `python3` machine. As long as it also has `python2` installed also.
* Travis make this `python3` conversion effort MUCH EASIER, because it gives INSTANT FEEDBACK to EVERY CODE CHANGE on ALL OPERATING SYSTEMS, ALL AUTH MECHANISMS, and ALL PYTHON VERSIONS.  
* Converting to `python3` is a must. There's no choice.  Python3 in 2020 or else the app does not live beyond 2020.  
* The changes needed are relatively small.
* Tools such as `2to3`, `futurize`, and `modernize`, make it even easier.
* This conversion will happen. 
* Anyone can come and submit python3 patches, and Travis immediate feedback will show the validity and if the python3 conversion patches don't also break existing `python2`.

Reply:
We already got 3 attempts to make offlineimap support py3. All failed with critical issues. The required changes are big and introduce subtle bugs because of bytes and UTF-8.

I don't think the conversion will happen, actually. I think the alternative will be imapfw.